snarkos/ledger/server.rs

```diff
@@ -273,6 +279,11 @@ impl<N: Network> Server<N> {
         Ok(reply::json(&ledger.ledger.read().get_block(height).or_reject()?))
     }
 
+    /// Returns the program with the given id
+    async fn get_program(program_id: u32, ledger: Arc<Ledger<N>>) -> Result<impl Reply, Rejection> {
```

Comment:
```suggestion
    async fn get_program(program_id: ProgramID<N>, ledger: Arc<Ledger<N>>) -> Result<impl Reply, Rejection> {
```

The program ID can't be represented with a u32.